cookbooks/kinematically_driven_subduction_2d/doc/kinematically_driven_subduction_2d.md

```diff
@@ -100,3 +99,98 @@ boundary. This can also be seen in {numref}`fig:QQ_case1_results`.
 
  Case 1 viscosity snapshots at 8 and 15 My.
 ```
+## Case 2a: Model with thermal evolution
+
+Case 2a builds on Case 1 by including a heterogeneous initial temperature field
+that combines the plate cooling model for the temperature in the plates with an
+adiabatic temperature gradient below the plates. For the plate cooling model,
+we assume that the overriding plate is 40 My old and the subducting plate 70 My.
+The temperature function is too complex to comfortably write out as a function
+expression in the input file, so it is put in a new initial temperature plugin
+called ’subduction plate cooling’. This plugin needs to be compiled as a shared
+library that can be called from the input file:
+
+```{literalinclude} Case2a_temperatures.prm
+
+```
+The initial temperature distribution can be seen in  {numref}`fig:QQ_case2a_temperature_t0`
+
+```{figure-md} fig:QQ_case2a_temperature_t0
+<img src="Case2a_temp_t0.*" />
+
+ Case 2a temperature distribution at time 0
+```
+
+The thermal expansivity is kept zero in Case 2a. Therefore the thermal evolution is
```

Comment:
The thermal expansivity is kept zero in Case 2a so that the thermal evolution...